source/features/bugs-tab.tsx

```diff
@@ -47,7 +85,9 @@ async function init(): Promise<void | false> {
 	// - update the count later
 	// On other pages:
 	// - only show the tab if needed
-	const isBugsPage = new SearchQuery(location.search).includes('label:bug');
+	const bugLabelKey = String(await cache.get(cacheKey()))!;
+	const bugLabel = bugLabelKey.includes(':') ? `"${bugLabelKey}"` : bugLabelKey;
```

Comment:
I'd like to see this turned into a helper inside SearchQuery itself, maybe a static method. Example:

``` JavaScript 
SearchQuery.fromEntries([
  ['label', label],
]) === 'label:"the bug"'
```

This would also work with multiple entries of course. It needs to use entries instead of an object because a query can have multiple `is` queries and could just be 

``` JavaScript 
SearchQuery.fromEntries([
  ['label', label],
  ['my other keyword']
]) === 'label:"the bug" my other keyword'
```

Reply:
I need a drop more guidance of how to implement.

Reply:
probably:

```js
entries.map(([base, value]) => {
  if (value === undefined) {
    return base
  }

  const escapedValue = /[:\s]/.test(value) ? `"${value}"` : value;
  return base + ':' + escapedValue
})
return entries.join(' ')

Reply:
Did I do it right?